package-lock.json

```diff
@@ -24,19 +24,6 @@
         "js-tokens": "^4.0.0"
       }
     },
-    "@kyleshockey/js-yaml": {
```

Comment:
We are not using package-lock files when developing modules to ensure that we always test the module against the latest dependency versions available, to reproduce the way how most of our users will install the module.

Because package-lock files are not published to npm, if there is a lock file in the project, then we will often end up with local dependencies being outdated and thus not able to reproduce problems reported by users that have installed the latest versions of our dependencies.

The other alternative is to setup a service like RenovateBot to keep the package-lock file up-to-date. We do that in loopback-next repo and I am quite annoyed about the pollution of the git history with lock-update commits. See e.g. [here](https://github.com/strongloop/loopback-next/commits/master/packages/testlab).